frontend/README.md

```diff
@@ -0,0 +1,42 @@
+# Frontend
+
+## Getting Started
+
+### Local Certificate (Required)
+
+Frist make sure you have go installed:
+
+```
+sudo snap install --classic go
```

Comment:
We should encapsulate this into a Dockerfile to avoid installing things on the host system and so the envs are easier to manage. :thinking: What do you think?

Reply:
We'd still need to install the ca authority on the host machine for the web browser to pick it up. Otherwise you'd end up running the browser in docker.

Reply:
After some pretty unfruitful trials, I ended up in state where the `mkcert` stuff is done in docker, but installing the certs is manual. Not convinced that saving the installs on the host sys. is worth the trouble. :thinking: I'm down for the cert stuff as-is. :rocket: